docs/architecture/decisions/008-sbom-push-with-gitlab-token.md

```diff
@@ -0,0 +1,25 @@
+| Status   | Date       | Author(s)                            |
+|:---------|:-----------|:-------------------------------------|
+| Proposed | 2025-05-14 | [@lmphil](https://github.com/lmphil) |
+
+## Context
+
+Adding GitLab integration to the Dependency Track/Hyades project is under consideration, with a key aspect being the use
+ of GitLab job ID tokens (in JWT format) for authentication, in addition to the existing API key-based authentication.
+ This new authentication method is needed to provide a more streamlined and secure experience for users who are already
+ authenticated with GitLab, reducing the need for additional credentials and minimizing the administrative burden
+ associated with managing multiple authentication tokens.
```

Comment:
If I understood correctly, this is specifically for short-lived tokens issued by GitLab CI? It would be good to explicitly call out this use-case.